pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -448,6 +455,8 @@ func ControlPlaneUpgraded(c client.Client, m maintenance.Maintenance, upgradeCon
 	}
 	for _, c := range clusterVersion.Status.History {
 		if c.State == configv1.CompletedUpdate && c.Version == upgradeConfig.Spec.Desired.Version {
+			// send controlplane upgrade complete timestamp
+			metrics.UpdateMetricControlPlaneEndTime(time.Now(), upgradeConfig.Name)
```

Comment:
See my comment elsewhere in the review.
```suggestion
			metrics.UpdateMetricControlPlaneEndTime(time.Now().UTC(), upgradeConfig.Name)
```